### PR TITLE
Add php migrate blog post

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: site/_posts/2020-10-01-updating-crufty-php-application.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
+---
+layout: blog-post
+title: Update a line of business PHP application to use OAuth
```
Comment (dpolstra) [inline site/_posts/2020-10-01-updating-crufty-php-application.md:3]: Securing your legacy PHP business application with OAuth

File: site/_posts/2020-10-01-updating-crufty-php-application.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+The code is all available in a [FusionAuth GitHub repository](https://github.com/fusionauth/fusionauth-example-php-connector) so you can clone it and follow along if you'd like.
+
+## Prerequisites
+
+You'll need to have a few things set up before you get going.
+
+* A running instance of FusionAuth. If you don't have this, check out the [5 minute setup guide](/docs/v1/tech/5-minute-setup-guide).
+* A modern PHP. This code was tested with php 7.3.11.
+
+## Introducing the legacy app
```
Comment (dpolstra) [inline site/_posts/2020-10-01-updating-crufty-php-application.md:27]: Introducing the legacy PHP app with a homegrown auth system

File: site/_posts/2020-10-01-updating-crufty-php-application.md [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+
+Save the application, and then add a user who is registered to the application. Navigate to "Users" and add a user. Then go to the "Registrations" tab and register them to "The ATM" application so they can continue to log in and do their job.
+
+{% include _image.liquid src="/assets/img/blogs/upgrade-php-application/user-added-to-application.png" alt="A user has been registered to the legacy application." class="img-fluid" figure=false %}
+
+Finally, you need the tenant identifier. If you just installed FusionAuth there is only one tenant, but FusionAuth supports multiple tenants, as many as you want. Navigate to "Tenants" and copy the id of the "Default" tenant. 
+
+That's all for FusionAuth configuration. Now let's look at the PHP application and how you'll change it.
+
+## Updating the PHP app
```
Comment (dpolstra) [inline site/_posts/2020-10-01-updating-crufty-php-application.md:133]: Updating the PHP app to authenticate using OAuth

**Comment by voidmain** [comment]
Looks good. I just committed a few changes directly to the branch. One as a verbiage change and the other was a image fix. 

I definitely want to read the next blog post though! That's where the meat is. :)

**Pull request merged**